packages/core/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@department-of-veterans-affairs/component-library",
+  "description": "VA.gov component library. Includes React and web components.",
+  "version": "3.3.0",
+  "license": "MIT",
+  "scripts": {
+    "build": "webpack"
+  },
+  "main": "dist/main-bundle.js",
+  "module": "src/main.js",
+  "dependencies": {
+    "@department-of-veterans-affairs/react-components": "workspace:*",
+    "@department-of-veterans-affairs/web-components": "workspace:*"
+  },
```

Comment:
Getting these two packages bundled as one is the main goal of this PR. The webpack config makes everything a named export, and in a followup PR I'd like to modify it (possibly with the copy plugin) to move the babel-ified files from `react-components` into the build output here so we don't immediately have to change the `vets-website` imports that look like this:

```js
import AdditionalInfo from '@department-of-veterans-affairs/component-library/AdditionalInfo';
```